plugins/source/aws/resources/services/inspector2/coverages.go

```diff
@@ -0,0 +1,48 @@
+package inspector2
+
+import (
+	"context"
+	"github.com/apache/arrow/go/v14/arrow"
+	"github.com/aws/aws-sdk-go-v2/service/inspector2"
+	"github.com/aws/aws-sdk-go-v2/service/inspector2/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+)
+
+func Coverages() *schema.Table {
+	tableName := "aws_inspector2_coverages"
+	return &schema.Table{
+		Name: tableName,
+		Description: `https://docs.aws.amazon.com/inspector/v2/APIReference/API_ListCoverage.html
+The 'request_account_id' and 'request_region' columns are added to show from where the request was made.`,
+		Resolver:  fetchInspector2Coverages,
+		Transform: transformers.TransformWithStruct(&types.CoveredResource{}, transformers.WithSkipFields("ResourceId")),
+		Multiplex: client.ServiceAccountRegionMultiplexer(tableName, "inspector2"),
+		Columns: []schema.Column{
```

Comment:
We are missing the default columns (req_account_id and req_account_id) that we include in all tables that support cross account and cross region aggregation

https://github.com/cloudquery/cloudquery/blob/9fe1ce1c5008250d8282f1fa31b6071f68f4fd50/plugins/source/aws/resources/services/inspector2/findings.go#L25-L36

Reply:
Done.